### ISSUE TITLE
Orders filter mistake

### ISSUE THREAD

**Issue by eflowUser** [XiaoFaye/WooCommerce.NET#40]
Hi! 
First of all thanks for the wraper.
I'm trying to get some Orders by fitering it by date, state, and including an specific ID.

`  
         Dictionary<string, string> parametros = new Dictionary<string, string>();

```
        parametros.Add("per_page", "1");
        parametros.Add("order", "asc");
        parametros.Add("after", info.FechaInicio.ToString ("2016-10-30"));           
        parametros.Add("status", "completed");
        parametros.Add("include", "950");

        RestAPI restapi = new RestAPI(ApiUrl, ConsumerKey, ConsumerSecret);
        WCObject wc = new WCObject(restapi);
        OrderList orders = Task.Run(() => wc.GetOrders  (parametros)).Result;`
```

But the result is that I obtain 2 orders (instead of 1 "per_page"), with creation date before "2016-10-30", the status filter is ok, but it neither has included the order 950 (that exists).

I've tried to call GetOrders with only one parameter at time and the call ignore it all, except the "status" filter.

I've get the parameter names from [https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#list-all-orders](https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#list-all-orders)

Please, could you tell me what's wrong on my code?
Thanks in advance.

**Comment by XiaoFaye** [comment]
Can you confirm that you are NOT using legacy restful api?

**Comment by eflowUser** [comment]
I'm USING LEGACY because if don't I get the Exception: "You need to use the legacy WCObject on the legacy version of WooCommerce Restful API."

**Comment by XiaoFaye** [comment]
Please try not to use the legacy api as your filters are only for the latest version of REST API. You can check the project home page for sample code.

**Comment by eflowUser** [comment]
Well... I try NOT USING Legacy and then I get an error about date format

At [http://woocommerce.github.io/woocommerce-rest-api-docs](http://woocommerce.github.io/woocommerce-rest-api-docs) I could find difrent date formats but no one works:

- YYYY-MM-DD (ex: 2016-10-25) returns invalid date format {"after":"La fecha que has proporcionado no es v\u00e1lida."}
-YYYY-DD-MM returns the same
- YYYY-MM-DDTHH:MM:SSZ (ex: 2016-10-25T00:00:00Z) returns {"code":"woocommerce_rest_authentication_error","message":"Firma no v\u00e1lida - la firma proporcionada no coincide.","data":{"status":401}}
-YYYY-MM-DD HH:MM:SS  or YYYY-MM-DDTHH:MM:SS returns the same 

Pelase, what's the corret date format?

**Comment by XiaoFaye** [comment]
Hi, there is a bug in the wrapper and I have released a beta version 0.5.6, could you please update and try again? Date format YYYY-MM-DDTHH:MM:SSZ should work. Thanks.
